strongbox-web-core/src/main/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 server.port=48080
 spring.liquibase.enabled=false
+spring.main.allow-bean-definition-overriding=true
```

Comment:
@ptirador why we need this property?

Reply:
@sbespalov it's mentioned in the PR's description :) 

> This version [disables bean overriding](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding) by default, so setting `spring.main.allow-bean-definition-overriding` to `true` is needed.

Reply:
ok, I will ask another way :)
why do we need to enable bean definition overriding?

Reply:
If we don't enable this, the following error appears when executing integration tests:

APPLICATION FAILED TO START
***************************

Description:

The bean 'authenticationManager', defined in class path resource [org/carlspring/strongbox/configuration/StrongboxSecurityConfig.class], could not be registered. A bean with that name has already been defined in URL [jar:file:/home/nullptr/.m2/repository/org/carlspring/strongbox/strongbox-security-api/1.0-SNAPSHOT/strongbox-security-api-1.0-SNAPSHOT.jar!/org/carlspring/strongbox/security/managers/AuthenticationManager.class] and overriding is disabled.

Action:

Consider renaming one of the beans or enabling overriding by setting spring.main.allow-bean-definition-overriding=true

Reply:
ok 👍

Reply:
@ptirador can we resolve naming issue instead?

Reply:
I just realized that `org.carlspring.strongbox.security.managers.AuthenticationManager` is probably initialized twice. It is marked as `@Component` so it takes part in component scanning and next we try to initialize it explicitly in `org.carlspring.strongbox.configuration.StrongboxSecurityConfig#authenticationManager`.

This is a bug. It needs to be solved.

Reply:
I agree. @ptirador : Would you mind investigating this?

Reply:
I'm trying to solve this but I'm not able to. Could you help me, @fuss86?

Reply:
Have you tried erasing 

```
@Component
@Scope("singleton")
```
from `org.carlspring.strongbox.security.managers.AuthenticationManager` ?

Reply:
Thank you so much! I've solved that and a few more bean issues related.

Reply:
Glad I could help :)